ssh/connection.go

```diff
@@ -87,13 +87,25 @@ func (tr *SSHTransfer) IsMultiplexingEnabled() bool {
 
 // Connection returns the nth connection (starting from 0) in this transfer
 // instance or nil if there is no such item.
```

Comment:
It might be worth updating this comment to reflect the revised logic in the function, maybe something like:
```suggestion
// Connection returns the nth connection (starting from 0) in this transfer
// instance if it is initialized and otherwise initializes a new connection and
// saves it in the nth position.  In all cases, nil is returned if n is greater than
// the maximum number of connections.
```

Reply:
Squashed in with a co-author credit.

Reply:
> Squashed in with a co-author credit.

Reply:
]()